### PR TITLE
Create a README.md file

### PR DESCRIPTION
**Pull request by oleg-alexandrov**
Thank you for the very useful example. I added a little text to the readme, you are welcome to adjust it as you see fit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
-This repository describes how to create, build, and run a-Python based ROS package.
+This repository describes how to create, build, and run a Python-based ROS package.
 
 Here is how to build this software.
 
-mkdir simontest
-cd simontest
-mkdir src
-git clone https://github.com/SimonBirrell/simontest src/simontest
-catkin build
```
Comment (IoannisDadiotis) [inline README.md]: dfghjk

**Comment by andrejpan** [comment]
Would be nice to merge this!

**Review by andrejpan — APPROVED** [review]

**Review by SimonBirrell — APPROVED** [review]
Thanks! Sorry I missed this!

**Comment by andrejpan** [comment]
@SimonBirrell can you merge this?

**Comment by SimonBirrell** [comment]
Thanks!

**Pull request merged**